ini/stm32-common.ini

```diff
@@ -31,11 +41,15 @@ extra_scripts = ${common_stm32.extra_scripts}
                 pre:buildroot/share/PlatformIO/scripts/generic_create_variant.py
                     buildroot/share/PlatformIO/scripts/offset_and_rename.py
 
+[stm32_variant_latest]
+extends       = common_stm32_latest
+extra_scripts = ${stm32_variant.extra_scripts}
+
 #
 # USB Flash Drive mix-ins for STM32
 #
 [stm_flash_drive]
-platform_packages = framework-arduinoststm32@https://github.com/rhapsodyv/Arduino_Core_STM32/archive/usb-host-msc-cdc-msc-3.zip
+platform_packages = framework-arduinoststm32@https://github.com/quiret/Arduino_Core_STM32.git#usb-host-msc-cdc-msc-3
```

Comment:
We're using ZIP intentionally, to sidestep the requirement for users to take the extra step to install a git command line.

Reply:
OK! I will follow your guideline and change it to .zip.